k8s.gcr.io/lib.sh

```diff
@@ -191,6 +191,20 @@ function ensure_gcs_bucket() {
     gsutil iam ch allUsers:objectViewer "${bucket}"
 }
 
+# Sets the web policy on the bucket, including a default index.html page
+# $1: The GCP project
+# $2: The bucket
+function ensure_gcs_web_policy() {
+    if [ $# -lt 2 -o -z "$1" -o -z "$2" ]; then
+        echo "ensure_gcs_web_policy(project, bucket) requires 2 arguments" >&2
+        return 1
+    fi
+    project="$1"
```

Comment:
never usedf?

Reply:
So...

It is perhaps surprising that the bucket is globally named and that you don't need a project, and so in the GCS functions I started passing the project anyway for homogeneity with the GCR functions.

But, yes, it is perhaps a little silly.